FordLabs/PeopleMover
# Add margin to person container the covers the new icon

## Issue
Resolves #521  

## What was done
- [x] Added margin to person container to avoid new icon being cut off

## How to test
1. Add new icon to person at top of unassigned
2. Ensure the new icon is not cut off
3. Make sure spacing of the people container looks okay on products

### FOR DEVS
Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/

Feature Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20FeatureBranch/

### FOR APPROVAL
Approval URL: https://approvalbranchui.apps.pd01i.edc1.cf.ford.com/

Approval Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20ApprovalBranch/
